code/modules/mob/living/bloodcrawl.dm

```diff
@@ -1,30 +1,3 @@
-/obj/effect/dummy/phased_mob/slaughter //Can't use the wizard one, blocked by jaunt/slow
-	name = "water"
-	icon = 'icons/effects/effects.dmi'
-	icon_state = "nothing"
```

Comment:
are you dropping this?

Reply:
There is no "nothing" icon_state in `effects.dmi`, and the same icon file is used in`obj/effect` anyway.